src/styles/defaultTheme.js

```diff
@@ -22,8 +29,20 @@ export default {
             main: COLOR_WARNING,
         },
         background: {
-            primary: COLOR_WHITE,
-            secondary: '#f9fafc',
+            main: COLOR_WHITE,
+            secondary: darken(COLOR_WHITE, 0.2),
+        },
+        divider: darken(COLOR_WHITE, 0.12),
+        text: {
+            main: COLOR_DARK_1,
+            secondary: darken(COLOR_DARK_1, 0.3),
+            disabled: darken(COLOR_DARK_1, 0.5),
+        },
+        action: {
+            active: darken(COLOR_WHITE, 0.54),
+            hover: darken(COLOR_WHITE, 0.1),
+            selected: darken(COLOR_WHITE, 0.2),
+            disabled: darken(COLOR_WHITE, 0.3),
```

Comment:
default values shouldn't be computed because it will be a visual breaking change